shm/inc/Player.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "shm/inc/Cargo.hpp"
+#include "shm/inc/Ship.hpp"
```

Comment:
You can make a forward declaration and include both headers in .cpp file. This causes compilation faster because any changes in both files will not trigger rebuild this one.